models/tasktemplate/fromdirs_test.go

```diff
@@ -61,7 +61,7 @@ func TestLoadFromDir(t *testing.T) {
 	_, err = dbp.DB().Query("DELETE FROM task_template;")
 	assert.Nil(t, err, "Emptying database failed")
 
-	err = tasktemplate.LoadFromDir(dbp, "templates_tests")
+	err = tasktemplate.LoadFromDirs(dbp, "templates_tests:templates_additionnal_tests")
```

Comment:
you could split the paths here and call tasktemplate.LoadFromDir for each found path

Reply:
ack